### DISCUSSION TITLE
Simple Hooks example?

### DISCUSSION THREAD

**Discussion by Shafik23** [spinspire/pocketbase-sveltekit-starter#14] [General]
Love this repo, thanks again for putting it together!

Can you provide the simplest example possible of a Hook configured directly in the PocketBase admin UI?

**Comment by jkdoshi** [comment]
Glad you find it useful.

Here's a screenshot:
![image](https://user-images.githubusercontent.com/2412869/217122377-2a81ee20-6876-45c5-a66d-6987f13d2b75.png)

See the annotations in the screenshot.

Also, see the source code of the script `backend/example-hook-script.sh`. It will receive the JSON of the record that changed.

Let me know if you need more guidance.

**Comment by j2l** [comment]
Thanks @jkdoshi !
However on docker, I get error `exit status 127`.
And when executed manually within the container:
```
~ # ./example-hook-script.sh
env: can't execute 'bash': No such file or directory
~ # sh ./example-hook-script.sh
PARAMS=
./example-hook-script.sh: line 12: jq: not found
```

After commenting the last line of the script, it works locally, but when executed through the hook I get:
`command start failed: /bin/sh /app/pb/example-hook-script.sh fork/exec /bin/sh /app/pb/example-hook-script.sh: no such file or directory`

**Reply by jkdoshi** [reply]
It does work for me in a container. But there is a difference. I'm not using `sh /app/pb/example-hook-script.sh` as my action.

My action is just `/app/pb/example-hook-script.sh`. BUT the first line of `/app/pb/example-hook-script.sh` is this ...
```
#!/usr/bin/env bash
...
```

So the above script requires `bash` to be preinstalled. In case you don't have it installed, you can either install it or replace it with `sh`

And I also have execute permission set on the file.
![image](https://github.com/spinspire/pocketbase-sveltekit-starter/assets/2412869/74abdeb1-a7f0-4cdb-9840-528b86b5c6a1)

Use `chmod +x /app/pb/example-hook-script.sh` in case yours does't.

**Reply by j2l** [reply]
thank you @jkdoshi !
Yes, `#!/usr/bin/env sh` does the trick!
How would you install bash since we don't have apt installed in the container?

EDIT: `apk add jq` to install jq, same for bash
